lib/manager/tool-versions/extract.spec.ts

```diff
@@ -0,0 +1,146 @@
+import { Fixtures } from '../../../test/fixtures';
+import { extractPackageFile } from './extract';
+
+describe('manager/tool-versions/extract', () => {
+  describe('extractPackageFile()', () => {
+    it('returns null for empty', () => {
+      expect(extractPackageFile('', '.tool-versions', {})).toBeNull();
+    });
+
+    it('extracts dependencies for supported tools', () => {
+      const res = extractPackageFile(
+        Fixtures.get('.tool-versions'),
+        'unused_file_name',
+        {}
+      );
+      expect(res).toEqual({
+        deps: [
+          {
+            currentValue: '10.15.0',
+            datasource: 'npm',
+            depName: 'nodejs',
+          },
+          {
+            currentDigest: 'v1.0.2-a',
+            currentValue: 'ref:v1.0.2-a',
+            datasource: 'pypi',
+            depName: 'python',
+          },
+          {
+            currentDigest: '39cb398vb39',
+            currentValue: 'ref:39cb398vb39',
+            datasource: 'ruby-version',
```

Comment:
I don't think we support this in our ruby-version datasource. Is this referring to a git hash in the ruby source repo?

Reply:
Same here, won't work. `ruby-version` datasource only supports strict ruby versions.

Reply:
I've replaced `ruby-version` with `github-tags` for versions starting with `ref:`.